ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -384,6 +384,28 @@ CallbackReturn URPositionHardwareInterface::on_activate(const rclcpp_lifecycle::
     RCLCPP_FATAL_STREAM(rclcpp::get_logger("URPositionHardwareInterface"), e.what());
     return CallbackReturn::ERROR;
   }
+  RCLCPP_INFO(rclcpp::get_logger("URPositionHardwareInterface"), "Calibration checksum: '%s'.",
+              calibration_checksum.c_str());
+  // check calibration
+  // https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/c3378599d5fa73a261328b326392e847f312ab6b/ur_robot_driver/src/hardware_interface.cpp#L296-L309
+  if (ur_driver_->checkCalibration(calibration_checksum)) {
+    RCLCPP_INFO(rclcpp::get_logger("URPositionHardwareInterface"), "Calibration checked successfully.");
+  } else {
+    RCLCPP_ERROR_STREAM(rclcpp::get_logger("URPositionHardwareInterface"), "The calibration parameters of the "
+                                                                           "connected robot don't match the ones from "
+                                                                           "the given kinematics "
+                                                                           "config file. Please be aware that this can "
+                                                                           "lead to critical inaccuracies of tcp "
+                                                                           "positions. Use "
+                                                                           "the ur_calibration tool to extract the "
+                                                                           "correct calibration from the robot and "
+                                                                           "pass that into the "
+                                                                           "description. See "
+                                                                           "[https://github.com/UniversalRobots/"
+                                                                           "Universal_Robots_ROS_Driver#extract-"
+                                                                           "calibration-information] "
+                                                                           "for details.");
```

Comment:
The problem is the clang-format job from pre-commit won't like it as that. 
```suggestion
        RCLCPP_ERROR_STREAM(rclcpp::get_logger("URPositionHardwareInterface"),
                        // clang-format off
        "The calibration parameters of the connected robot don't match the ones from the given kinematics config file. "
        "Please be aware that this can lead to critical inaccuracies of tcp positions. Use the ur_calibration tool to "
        "extract the correct calibration from the robot and pass that into the description. See "
        "https://github.com/UniversalRobots/Universal_Robots_ROS_Driver#extract-calibration-information for details.");
    // clang-format on
```

would work